.github/workflows/develop_commit_notification.yml

```diff
@@ -29,5 +29,6 @@ jobs:
             "app_name": "oppiaserver-backup-migration",
             "maintenance_mode": "false",
             "sha": "${{ github.sha }}",
+            "is_manual_build": "false",
```

Comment:
Suggest renaming to build_was_manually_triggered (since the build itself runs from a script, it's the trigger that's manual). Think that would be OK?

Reply:
done